src/cpp/shared/game/game.h

```diff
@@ -102,11 +110,10 @@ class game {
 	char lastSaveFile[251];
 	char _12[100];
 	SMapHeader mapHeader;
-	int field_44D;
-	__int16 field_451;
+	char relatedToPlayerPosAndColor[6];
 	char playerHandicap[6];
-	char field_459[6];
-	char field_45F[6];
+	char relatedToColorOfPlayerOrFaction[6];
```

Comment:
Somewhere there is a "#define NUM_PLAYERS 6". Use it.

Reply:
By "somewhere," I mean "at the top of this file."

Reply:
Got it. Fixed that bit.